GeosCore/aerosol_mod.F90

```diff
@@ -280,8 +282,13 @@ SUBROUTINE AEROSOL_CONC( Input_Opt,  State_Chm, State_Diag, &
     ENDIF
 
     IF ( RC /= GC_SUCCESS ) RETURN
+#if !defined( MODEL_BCC )
 #if !defined( MODEL_CESM )
     CALL HCO_GC_EvalFld( Input_Opt, State_Grid, Trim(FieldName), State_Chm%OMOC, RC, FOUND=FND )
+#else
+    FND = .True.
+    RC  = GC_SUCCESS
+#endif
```

Comment:
@sunrz7: I think we can simplify the logic here:
```F90
#if !defined( MODEL_BCC ) && !defined( MODEL_CESM )
   CALL HCO_GC_EvalFld( Input_Opt, State_Grid, Trim(FieldName), State_Chm%OMOC, RC, FOUND=FND )
#else
    FND = .True.
    RC  = GC_SUCCESS
#endif
```
Tagging @lizziel for a second opinion.